airbyte-cdk/python/unit_tests/sources/file_based/scenarios/scenario_builder.py

```diff
@@ -154,6 +158,10 @@ def set_expected_read_error(self, error: Type[Exception], message: str) -> "Test
         self._expected_read_error = error, message
         return self
 
+    def set_log_levels(self, levels: Set[str]) -> "TestScenarioBuilder":
```

Comment:
allow test developers to configure which logs are captured so we can verify the output of a read